pkg/apis/v1alpha1/machine.go

```diff
@@ -70,3 +71,20 @@ type MachineList struct {
 	metav1.ListMeta `json:"metadata,omitempty"`
 	Items           []Machine `json:"items"`
 }
+
+// FromNode Converts a node into a pseudo-Machine using known values from the node
+func FromNode(node *v1.Node) *Machine {
```

Comment:
In theory, this package could hold multiple APIs (if the versions lined up)
```suggestion
func MachineFromNode(node *v1.Node) *Machine {
```